script.js

```diff
@@ -1,406 +1,193 @@
-/*Here we have created two different arrays that you can work with if you want.
-If you choose to create your own arrays with elements, just make sure that some
-of the properties make sense to filter on, and some to sort on.*/
-
-const books = [
+const movies = [
```

Comment:
Great job! Good descriptions and easy to follow, well structured! /Kasia, Cholpon, Nella